server/tests/api/role.spec.js

```diff
@@ -0,0 +1,169 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../../app';
+import userData from '../testData/userData';
+import roleData from '../testData/roleData';
+
+const { superadmin, admin, author } = userData;
+const { role5, role6, role7, role8 } = roleData;
+
+let authorToken, adminToken, superadminToken;
+
+const expect = chai.expect;
+chai.use(chaiHttp);
+
+describe('Roles', () => {
+  before((done) => {
+    chai.request(server)
+      .post('/users/login')
+      .send(superadmin)
+      .end((err, res) => {
+        superadminToken = res.body.token;
+        done();
+      });
+  });
+
+  before((done) => {
+    chai.request(server)
+      .post('/users/login')
+      .send(admin)
+      .end((err, res) => {
+        adminToken = res.body.token;
+        done();
+      });
+  });
+
+  before((done) => {
+    chai.request(server)
+      .post('/users/login')
+      .send(author)
+      .end((err, res) => {
+        authorToken = res.body.token;
+        done();
+      });
+  });
+
+  // GET /roles
+  describe('GET /roles', () => {
+    it('should allow superadmin to view roles available', (done) => {
+      chai.request(server)
+        .get('/roles')
+        .set('x-access-token', superadminToken)
+        .end((err, res) => {
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.an('array').that.have.lengthOf(5);
+          expect(res.body[0]).to.have
+            .keys('id', 'title', 'createdAt', 'updatedAt');
+          done();
+        });
+    });
+
+    it('should deny access when others try to view roles', (done) => {
```

Comment:
I think this test shows that only an admin user is denied access to view roles, and not "other users" like it is labelled.